website/scripts/build-static-content.js

```diff
@@ -9,10 +9,11 @@ module.exports = {
 
   inputs: {
     dry: { type: 'boolean', description: 'Whether to make this a dry run.  (.sailsrc file will not be overwritten.  HTML files will not be generated.)' },
+    local: { type: 'boolean', description: 'Whether to send requests to the GitHub API when running this script. If this option is set to `true`, query contributer profiles will not be populated'},
```

Comment:
Can we:
1. name this `--dev` instead of `--local`
2. describe it as whether to "minimize requests to the GitHub API which usually can be skipped during local development, such as requests used for fetching GitHub avatar URLs"

Reply:
@mikermcneil the `--dev` flag already exists in Sails and logs this message when it is used: 
```
`--dev` option is deprecated: Please do not use it.
```

https://github.com/balderdashy/sails/blob/21b874b787138254ba1b878c6f29c1722b8f16a1/lib/app/configuration/load.js#L119-L123

Reply:
@eashaw It still works when used with sails run, but agreed the deprecation message is ugly.  How about `--no-github-avatars`, or something else?  Up to you, not a huge deal.   I wanted to avoid `--local` because it might create confusion for folks about local.js

Reply:
I renamed it to skipGithubRequests

Reply:
@eashaw Love the name

Reply:
Usage: 

```sh
sails run build-static-content --skipGitHubRequests
```